unit_tests/engine/test_rule_loader.cpp

```diff
@@ -43,6 +44,66 @@ class engine_loader_test : public ::testing::Test {
 		return ret;
 	}
 
+	// This must be kept in line with the (private) falco_engine::s_default_ruleset
+	uint64_t num_rules_for_ruleset(std::string ruleset = "falco-default-ruleset")
+	{
+		return m_engine->num_rules_for_ruleset(ruleset);
+	}
+
+	bool has_warnings()
+	{
+		return m_load_result->has_warnings();
+	}
+
+	bool check_warning_message(std::string warning_msg)
+	{
+		if(!m_load_result->has_warnings())
+		{
+			return false;
+		}
+
+		for(auto &warn : m_load_result_json["warnings"])
+		{
+			std::string msg = warn["message"];
+			// Debug:
```

Comment:
leftover :)

Reply:
actually, this was intentional, in this way when we want to debug the errors or warnings we can simply uncomment the line below. AFAIK gtest has no cool features to enable logs only when the test fails, while this is the default for other languages like Rust :(

Reply:
i can remove it if you want, again no strong opinion here

Reply:
It's ok, we can leave it here!